evaluator/evaluate.go

```diff
@@ -53,6 +54,7 @@ func configure() {
 	enableRepoAnalysis = utils.GetBoolEnvOrDefault("ENABLE_REPO_ANALYSIS", true)
 	enableStaleSysEval = utils.GetBoolEnvOrDefault("ENABLE_STALE_SYSTEM_EVALUATION", true)
 	enableLazyPackageSave = utils.GetBoolEnvOrDefault("ENABLE_LAZY_PACKAGE_SAVE", true)
+	enablePrunePackageLatestOnly = utils.GetBoolEnvOrDefault("ENABLE_UPDATES_LATEST_ONLY", false)
```

Comment:
Could you keep consistent go variable and env variable name, pls?

Reply:
Fixed